src/GameLoop.tsx

```diff
@@ -55,223 +59,214 @@ const stateMachine = createMachine({
 
 // Handle states.
 const stateHandler = interpret(stateMachine).onTransition((state) => {
-    // TODO
+    console.log("   STATE:", state.value);
 });
 stateHandler.start();
 
-let placedCells = new Set();
+// Various game logic vars.
+const FPS =
+    60; /* Note: with 60 FPS, this is a float (16.666..7). Might run into issues. */
+const frameStep = 1000 / FPS;
+let accumFrameTime = 0;
+let prevFrameTime = performance.now();
+const placedCells =
+    new Set(); /* Block cell coordinates that were placed on the current frame. */
```

Comment:
For comments like this for a variable/global, if it's more than just a short phrase, I'd put it on its own line above the variable, and then add empty lines around the comment and variable.

Reply:
Will do